index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>MrkAmin Portfoli</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <header>
+          <div class="logo-container">
+            <p>Mohammad</p>
+          </div>
+          <div class="menu">
+            <img src="./image/menu-icon.png" alt="Menu Icon" />
+          </div>
+          <div class="Group">
+            <img src="./image/Group 4.png" alt="Menu Icon" />
+          </div>
+          <div class="time-style">
+            <img src="./image/Time Style.png" alt="Menu Icon" />
+          </div>
+        </header>
+        <div class="home">
+          <h1>Hey There.<br />I'm Mohammad Rafi Amin</h1>
+          <h2>I'm a software Developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don't hestiate to contact me.
+          </p>
+          <div class="social">
+            <div class="social-icons">
+              <div class="icon">
+                <img
+                  src="./image/github-icon.png"
+                  alt="Github icon"
+                  class="icons"
+                />
+              </div>
+              <div class="icon">
+                <img
+                  src="./image/linkedin-icon.png"
+                  alt="Linkedin icon"
+                  class="icons"
+                />
+              </div>
+              <div class="icon">
+                <img
+                  src="./image/chill-icon.png"
+                  alt="Angelist Icon"
+                  class="icons"
+                />
+              </div>
+              <div class="icon">
+                <img
+                  src="./image/medium-icon.png"
+                  alt="Medium Icon"
+                  class="icons"
+                />
+              </div>
```

Comment:
I kindly suggest you to use UL/LI lists to organize your social icons

Reply:
@mgmediaweb
Thanks for your good and valuable feedback.